org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ScalaProject.scala

```diff
@@ -589,7 +589,11 @@ class ScalaProject private(val underlying: IProject) extends ClasspathManagement
         if (!storage.getBoolean(useSbtCompilerProperty))
           new SbtScopesBuildManager(this, settings)
```

Comment:
ok for doing it an extension,
going back to described problems with cleaning two dependent project. now I suffer with NPE in SbtBuild.sbtClientWatcher#135. 'system' is null and means that SbtRemotePlugin.system is null in such scenario. I look at this on Monday